tests/integration/MyObservations.test.js

```diff
@@ -11,57 +11,80 @@ import factory, { makeResponse } from "../factory";
 import { renderAppWithComponent } from "../helpers/render";
 import { signIn, signOut } from "../helpers/user";
 
-jest.useFakeTimers( );
+jest.useFakeTimers();
 
-describe( "MyObservations", ( ) => {
-  beforeEach( signOut );
+jest.mock( "i18next", () => {
+  const originalModule = jest.requireActual( "i18next" );
 
-  afterEach( ( ) => {
-    jest.clearAllMocks( );
-  } );
+  return {
+    // Ideally we shouldn't have to test i18next itself
+    // just need to test that the correct keys/opts are being fed to i18next
+    t: ( key, options ) => {
+      if ( !options ) {
+        return key;
+      }
+      return originalModule.t( key, options );
+    }
+  };
+} );
 
-  it( "should not have accessibility errors", async ( ) => {
-    const mockUser = factory( "LocalUser" );
-    await signIn( mockUser );
-    const observations = [factory( "RemoteObservation" )];
-    inatjs.observations.search.mockResolvedValue( makeResponse( observations ) );
-    const { queryByTestId } = renderAppWithComponent( <ObsList /> );
-    await waitFor( ( ) => {
-      expect( queryByTestId( "ObservationViews.myObservations" ) ).toBeAccessible( );
-    } );
+describe( "MyObservations", () => {
+  beforeEach( signOut );
+
+  afterEach( () => {
+    jest.clearAllMocks();
   } );
 
-  describe( "when signed out", ( ) => {
-    async function testApiMethodNotCalled( apiMethod ) {
-      const signedInUsers = global.realm.objects( "User" ).filtered( "signedIn == true" );
+  // Hacky solution, but making the test for signed out users first
+  // signIn has some side-effects that aren't getting properly cleaned up
+  // Users persist somehow while the test is running, despite being removed
+  // from the global realm instance
+  describe( "when signed out", () => {
+    async function testApiMethodNotCalled( apiMethod, realm ) {
+      const signedInUsers = realm.objects( "User" );
       expect( signedInUsers.length ).toEqual( 0 );
+
       const { getByText } = renderAppWithComponent( <ObsList /> );
-      await waitFor( ( ) => {
-        expect( getByText( "Log in to iNaturalist" ) ).toBeTruthy( );
+      await waitFor( () => {
+        expect( getByText( "Log-in-to-iNaturalist" ) ).toBeTruthy();
```

Comment:
Test the key got returned. In the test suite there was a weird issue where the first test, the `t` func would return the key, but the second test would return the translated text. Seemed simpler to test that the correct key is getting fed to the component, rather than testing the i18next library.